scripts/qa_crowbarsetup.sh

```diff
@@ -985,6 +985,17 @@ EOF
         rm -f $pfile
         crowbar ntp proposal commit default
     fi
+    if [[ $keep_existing_hostname = 1 ]] ; then
```

Comment:
The problem is the proposal step is to late, we must change the keep_existing_hostname at the end of the instcrowbar step.